Mesh.h

```diff
@@ -0,0 +1,87 @@
+/*
+ @file      Mesh.h
+ @brief     contains Mesh class
+ @author    Luke Eure
+ @date      January 9 2016
+*/
+
+#ifndef MESH_H
+#define MESH_H
+
+#include <iostream>
+#include <vector>
+#include <math.h>
+
+#include "Material.h"
+#include "Boundaries.h"
+
+class Mesh {
```

Comment:
Pretty nice class... only bummer is that it locks you into box geometries... but I guess that's fine for a simple application.